### ISSUE TITLE
Example is invalid

### ISSUE THREAD

**Issue by Disinterpreter** [vsTerminus/Mojo-Discord#1]
Example in README.md  is invalid.

```
Can't locate object method "connect" via package "Mojo::Discord" at
```

**Comment by vsTerminus** [comment]
Yea - my bad. The example in the README hasn't been updated to reflect recent changes.

I'm currently working on making the library more state-aware (things like users, roles, channels, etc) and will update the example once that is functional.

In the meantime, you can check out my [Discord-NP](https://github.com/vsTerminus/Discord-NP/) and [Goose](https://github.com/vsTerminus/Goose) repos for working examples.

**Comment by Disinterpreter** [comment]
@vsTerminus ok, just now i'm write a bot on a Perl, and want more functionality, f.e work with audio :D

**Comment by vsTerminus** [comment]
Fixed in a3b4905
